spec/titleCaseSpec.js

```diff
@@ -0,0 +1,16 @@
+
+describe('applying titleCase to a fragment', function() {
+var titleCase = require('../helpers/titleCase')
```

Comment:
Very minor niggle, but I think this line should be indented for consistency

Reply:
Yes, you're right, indented now.